vm/src/hint_processor/builtin_hint_processor/uint512_utils.rs

```diff
@@ -0,0 +1,95 @@
+use crate::Felt252;
+use crate::{
+    hint_processor::hint_processor_definition::HintReference,
+    serde::deserialize_program::ApTracking,
+    stdlib::{borrow::Cow, boxed::Box, collections::HashMap, prelude::*},
+    types::relocatable::Relocatable,
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+};
+use num_bigint::BigUint;
+
+use super::{
+    hint_utils::get_relocatable_from_var_name,
+    uint_utils::{pack, split},
+};
+
+pub(crate) struct Uint512<'a> {
```

Comment:
Please remove the whole file, as the `Uint512` type is already defined at vm/src/hint_processor/builtin_hint_processor/secp/bigint_utils.rs

Reply:
This was part of the #1399 refactor

Reply:
Oh nice. Will do.